### PR TITLE
Feat(kubernetes): documentation about kubernetes integration using th…

### PR DESCRIPTION
**Pull request by leleueri**
…e gravitee-gateway-services-kuberentes

Fixes gravitee-io/issues#5173
Fixes gravitee-io/gravitee-docs#379

### CODE REVIEW

File: pages/apim/3.x/kubernetes/apim-kubernetes-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+= Kubernetes Plugin
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_overview.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+:page-liquid:
+
+== Overview
+
+APIM 3.7.0 introduces a kubernetes plugin for the Gateway allowing the deployement of APIs using https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom Resource Definitions].
```
Comment (brasseld) [inline pages/apim/3.x/kubernetes/apim-kubernetes-overview.adoc:11]: `allowing the deployment of APIs`

File: pages/apim/3.x/kubernetes/apim-kubernetes-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+APIs deployed using CRDs are not visible through the APIM Management Console.
+
+WARNING: This plugin is currentlty in Alpha version, the Api-Key policy isn't available for APIs deployed using this plugin.
```
Comment (brasseld) [inline pages/apim/3.x/kubernetes/apim-kubernetes-overview.adoc:15]: This plugin is currently in Alpha version

**Review by brasseld — CHANGES_REQUESTED** [review]

File: pages/apim/3.x/kubernetes/apim-kubernetes-installation.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+:page-permalink: apim/3.x/apim_kubernetes_installation.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+:page-liquid:
+
+== Create Custom Resource Definitions (CRDs)
+
+The Kubernetes plugin comes with three CRDs that must be created using `kubectl`:
+
+* https://raw.githubusercontent.com/gravitee-io/gravitee-gateway/master/gravitee-gateway-services/gravitee-gateway-services-kubernetes/src/main/crds/gravitee-plugin-crd.yml[GraviteePlugins]
```
Comment (plsstephenson) [inline pages/apim/3.x/kubernetes/apim-kubernetes-installation.adoc:13]: These links don't work

**Review by plsstephenson — CHANGES_REQUESTED** [review]

File: pages/apim/3.x/kubernetes/apim-kubernetes-installation.adoc (hunk 3 above)
Comment (leleueri) [reply pages/apim/3.x/kubernetes/apim-kubernetes-installation.adoc]: Links are fixed.

Comment (leleueri) [reply pages/apim/3.x/kubernetes/apim-kubernetes-installation.adoc]: Links are fixed.

**Pull request merged**